redfish-core/lib/assembly.hpp

```diff
@@ -195,6 +195,50 @@ inline void
                                 "LocationCode",
                                 "LocationCode");
                         }
+                        else if (interface == "xyz.openbmc_project.State."
+                                              "Decorator.OperationalStatus")
+                        {
+                            crow::connections::systemBus->async_method_call(
+                                [aResp, assemblyIndex](
+                                    const boost::system::error_code ec,
+                                    const std::variant<bool>& property) {
+                                    if (ec)
+                                    {
+                                        BMCWEB_LOG_DEBUG
+                                            << "DBUS response error";
+                                        messages::internalError(aResp->res);
+                                        return;
+                                    }
+
+                                    nlohmann::json& assemblyArray =
+                                        aResp->res.jsonValue["Assemblies"];
+                                    nlohmann::json& assemblyData =
+                                        assemblyArray.at(assemblyIndex);
+
+                                    // Set the default Health to OK
+                                    assemblyData["Status"]["Health"] = "OK";
```

Comment:
Do we want a "Status""State" ?
Should we move this outside of this else if so we could have a health of OK if this interface doesn't exist? 
Maybe set 
"Status""State" = "Enabled" 
"Status""Health" = "OK"
outside of this else if?

Reply:
Done